R/finalize_country_name.R

```diff
@@ -0,0 +1,52 @@
+#### finalize_country_name ####
+
+#' @name finalize_country_name
+#' @title Chooses the country_name, that is most likely to be correct 
+#'
+#' @description Picks the country name in a hierarchical order from the results
+#' of the determine_country_by_coords and standardize_country_name functions, 
+#' followed by the original input of the database
+#'
+#' @param x an object of class c14_date_list
+#'
+#' @return an object of class c14_date_list
+#' @export
+#'
+#' @rdname finalize_country_name
+#'
+finalize_country_name <- function(x) {
+  UseMethod("finalize_country_name")
+}
+
+#' @rdname finalize_country_name
+#' @export
+finalize_country_name.default <- function(x) {
+  stop("x is not an object of class c14_date_list")
+}
+
+#' @rdname finalize_country_name
+#' @export
+finalize_country_name.c14_date_list <- function(x) {
+  
+  # check if the columns country_coord and country_thes are present
+  necessary_vars <- c("country_coord", "country_thes")
+  x %>% check_if_columns_are_present(necessary_vars)
```

Comment:
there is something missing here; where does the pipe lead to?

Reply:
I was hoping, that if the check was positive it wouldn´t have to lead anywhere and just move on and if the test is negative the function would be stopped by the check